inst/templates/tiltWorkflows.Rmd

```diff
@@ -1,236 +1,207 @@
+## Sector profile
 
 ```{r}
-sector_profile <- profile_sector(sp_company,
-  sp_scenarios,
+# Load data specific to this indicator
+sector_profile_companies <- read_csv(toy_sector_profile_companies())
+sector_profile_any_scenarios <- read_csv(toy_sector_profile_any_scenarios())
+
+# Create results at product and company level
+sector_profile <- profile_sector(
+  sector_profile_companies,
+  sector_profile_any_scenarios,
   europages_companies = europages_campanies,
-  ecoinvent_activities = ei_activities_ov,
-  ecoinvent_europages = mapper_ep_ei,
-  isic_tilt = isic_4digit_name,
+  ecoinvent_activities = ecoinvent_activities,
+  ecoinvent_europages = ecoinvent_europages,
+  isic_tilt = isic_tilt,
   low_threshold = 1 / 3,
   high_threshold = 2 / 3
 )
 
-sector_profile_at_product_level <- sector_profile |> unnest_product()
-sector_profile_at_company_level <- sector_profile |> unnest_company()
-```
+sector_profile_at_product_level <- sector_profile |> 
+  unnest_product()
+sector_profile_at_product_level
 
-### Save the Sector profile final results
+sector_profile_at_company_level <- sector_profile |> 
+  unnest_company()
+sector_profile_at_company_level
 
-```{r}
 sector_profile_at_product_level |>
-  write_csv("/path/sector_profile_at_product_level.csv")
-
+  write_csv(tilt_path("sector_profile_at_product_level.csv"))
 sector_profile_at_company_level |>
-  write_csv("/path/sector_profile_at_company_level.csv")
+  write_csv(tilt_path("sector_profile_at_company_level.csv"))
 ```
 
 ## Sector profile upstream
 
-### Load Data
-
-The input data of Sector profile upstream which comes from tiltIndicatorBefore
-package becomes the input for the tiltIndicator package.
-
 ```{r}
-spu_company <- read_csv("/path/sector_profile_upstream_companies.csv")
-
-spu_product <- read_csv("/path/sector_profile_upstream_products.csv")
-
-spu_scenarios <- read_csv("/path/sector_profile_any_scenarios.csv")
-```
-
-### Create Sector profile upstream product and company level outputs.
-
-The code takes the output of tiltIndicator and use it as an input for the
-tiltIndicatorAfter package.
-
-```{r}
-sector_profile_upstream <- profile_sector_upstream(spu_company,
-  spu_scenarios,
-  spu_product,
+# Load data specific to this indicator
+sector_profile_upstream_companies <- read_csv(toy_sector_profile_upstream_companies())
+sector_profile_upstream_products <- read_csv(toy_sector_profile_upstream_products())
+
+# Create results at product and company level
+sector_profile_upstream <- profile_sector_upstream(
+  sector_profile_upstream_companies,
+  sector_profile_any_scenarios,
+  sector_profile_upstream_products,
   europages_companies = europages_campanies,
-  ecoinvent_activities = ei_activities_ov,
-  ecoinvent_inputs = ei_input,
-  ecoinvent_europages = mapper_ep_ei,
-  isic_tilt = isic_4digit_name,
+  ecoinvent_activities = ecoinvent_activities,
+  ecoinvent_inputs = ecoinvent_inputs,
+  ecoinvent_europages = ecoinvent_europages,
+  isic_tilt = isic_tilt,
```

Comment:
It's best to be consistent with the names of the arguments. That way our readers will gradually learn the names they'll see everywhere.